README.md

```diff
@@ -49,8 +49,23 @@ const BigNumber = require('bignumber.js');
 
 ES6 module:
 
+```bash
+$ npm install bignumber.js --save
+```
+
+```javascript
+import BigNumber from "bignumber.mjs"
+```
+
+Typescript:
+
+```bash
+$ npm install bignumber.js --save
+$ npm install @types/bignumber.js --save-dev
```

Comment:
There is no need to install the typings. They are already present in the bignumber.js install.